pubspec.yaml

```diff
@@ -5,8 +5,8 @@ homepage: https://github.com/ubuntu/yaru_widgets.dart
 version: 1.1.3
 
 environment:
-  sdk: ">=2.17.0 <3.3.2"
-  flutter: ">=3.3.2"
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=3.3.0"
```

Comment:
people are strongly encouraged to upgrade to 3.3.2 because of the important bug fixes but technically (API-wise) only 3.3.0 is required